src/main/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<configuration>
+    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>[%d] %p %m \(%c:%t\)%n</pattern>
```

Comment:
I like the long format specifiers better (%date, %message, ...) because otherwise I always have to look up the meaning.

Also, I would align and length limit fields such as level and class.

Can you include some sample logging output?

Appending the logger name and the thread in the end in the format given is quite unusual. Isn't it? The meaning is not the same as in the log4j configuration.